### DISCUSSION TITLE
Sales Contact

### DISCUSSION THREAD

**Discussion by tkiblin** [moosefs/moosefs#553] [General]
Hey MooseFS team, sorry to post here but I've tried the online web contact form, emailing directly and even opening up a support ticket - and so far no response.

I'd like to speak with someone in sales about a commercial license for an opportunity we are working on.

**Comment by chogata** [comment]
It's rare, but sometimes our ticketing system goes crazy and hides e-mails. I forwarded this to our sales department, they will look for your messages. If they cannot find them, someone will contact you here.

**Reply by tkiblin** [reply]
Thank you - here is the ticket # - sales contact [Support #7408]

**Reply by chogata** [reply]
I'm told someone replied to you, did you get it?

**Comment by dragon2611** [comment]
They do seem to want to make it hard, did try filling in the contact form and later emailing for a quote for a small cluster as I'd be interested in the Failover support for the master.

So far heard exactly nothing back, no idea if they even got the enquiry or just aren't interested. 

I actually really hate the whole "Contact us for pricing" approach to things, I can get that for large/custom deployments it may be beneficial but why can't companies just publish pricing online it makes things so much easier particularly when researching as to if something fits within a budget.

**Reply by pkonopelko** [reply]
Hi @dragon2611, could you please email me directly at peter@mfs.io? Thanks

**Reply by dragon2611** [reply]
Email sent, thanks.

**Reply by tkiblin** [reply]
Eventually after posting on here I got the "we never got it" reply.  I tried to get pricing but after a couple of weeks I gave up and moved on - when you say "Can I get budgetary pricing on this capacity" and never hear back, well, if sales can't / doesn't want to sell, you have to worry about how good support would be and the longevity of the company imo.

If you are looking for a small cluster (1PB or under), save yourself the time and frustration in trying to get or understand pricing and move on.

**Reply by dragon2611** [reply]
I feel they're missing a trick by not having an off the shelf licence option for people who want some of the pro features.

By all means they can have better rates for larger customers, that's pretty common.

Could be X per TB for PRO
X + Y for PRO and Support.etc

Similar to how Proxmox does their pricing depending on how much support coverage you want.

**Reply by pkonopelko** [reply]
@dragon2611 We received the email you sent yesterday to us, and I also got your today's bump to me directly (thank you), and I have just sent you a response. As I stated, I'll get back to you later today with details. Thanks!

@tkiblin I remember I was in touch with you myself. I was unaware you didn't get my response! Let me look into that. I am truly sorry for the inconvenience.

**Reply by dragon2611** [reply]
Hi @pkonopelko The email I sent yesterday was after filling the enquiry form a couple of days ago and hearing nothing, since there's not even an auto responder to confirm receipt I wasn't sure if the contact form was even working.

With the push for DKIM/SFP/DMARC.etc it's not uncommon for people to forget about the website forms when setting these up and thus breaking them in the process.

**Reply by pkonopelko** [reply]
Hi @dragon2611 We will review this matter carefully, and discuss possible changes in our systems. Your feedback is very valuable to us. We are committed to improving our services based not only on our clients', but also on all MooseFS Users experiences and suggestions. Thank you so much for your input!

**Reply by pkonopelko** [reply]
By the way, I have checked and it seems that on my GitHub profile no email address was visible. I didn't know that.

I've just added my @moosefs.com email address so that in case anyone encounters any issues with reaching out to us, can always find it easily and contact me directly. Hope it helps! Thanks again, cheers

**Reply by tianon** [reply]
> you have to worry about how good support would be and the longevity of the company imo

In the hopes that it helps allay some doubts, I can provide my own related anecdotes here; to be clear, I am not and have not ever been a paid customer of MooseFS (although I have considered it a few times both personally and professionally over what's now been many years).

Over those many years (itself speaking to the longevity of the company), I have reached out to their support several times, despite not being a paying customer.  Every time, they have been very prompt and helpful, never once even suggesting that they could help me better if I paid them or similar.  They're always polite, on point, and extremely knowledgeable about their product (even and especially the open source version).

Technical glitches, _especially_ related to email, happen to the best of us and IMO/IME, the MooseFS folks have even publicly proven time and again that they're extremely deserving of leeway here (case in point, Peter's own direct email address now publicly posted as an open invitation for escalation on emails falling to be delivered/responded to). ❤️

**Reply by dragon2611** [reply]
Still not heard anything other than an acknowledgement from @pkonopelko , not sure if they did send it, and it got eaten (although I can't find anything in the mail quarantine.etc) or they haven't sent it.

I've been testing using MFS for proxmox storage, and it seems to be working quite well so far but the problem I have at the moment is if the master goes down then none of the VM's on MFS would be able to read/write from their disks.

**Comment by anon314159** [comment]
Same issue.  RFQ's seem to go completely unanswered :-(

**Comment by wilhil** [comment]
Hi,

We are still using MooseFS open source and really want pro - we've reached out numerous times over the past year and haven't heard back.

I've also tried the contact form on the website and now getting "There was an error trying to send your message. Please try again later."

Are you still selling and supporting pro or should we go down a different route?

**Reply by pkonopelko** [reply]
Hello @wilhil thanks for letting us know. The error on our website is most probably related to the ReCaptcha mechanism. Can you please email me directly at piotr.konopelko@moosefs.com? Thanks
